JenkinsfileRelease

```diff
@@ -0,0 +1,80 @@
+pipeline {
+
+  parameters {
+    gitParameter name: 'branch',
+                description: 'The branch to check out',
+                branchFilter: 'origin/(.*)',
+                defaultValue: 'master',
+                selectedValue: 'DEFAULT',
+                type: 'PT_BRANCH',
+                listSize: '0',
+                quickFilterEnabled: true
+
+    string name: 'ReleaseVersion',
+         description: 'Version to release',
+         defaultValue: ''
+  }
+
+  agent {
+    label 'graphdb-jenkins-node'
+  }
+
+  options {
+    disableConcurrentBuilds()
+    timeout(time: 15, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    CI = "true"
+    NPM_TOKEN = credentials('npm-token')
+  }
+
+  stages {
+    stage ('Prepare') {
+      steps {
+        // Switch to branch
+        sh "git checkout ${branch}"
+
+        // Change versions
+        sh "npm version --allow-same-version --git-tag-version=false ${ReleaseVersion}"
+        dir("test-cypress/") {
+          sh "npm version --allow-same-version --git-tag-version=false ${ReleaseVersion}"
+        }
+
+        // Install
+        sh "npm ci"
+
+        // Build
+        sh "npm run build"
+      }
+    }
+
+    stage ('Publish') {
+      steps {
+        // Publish on npm
+        sh "echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc && npm publish"
+        // Publish cypress tests on npm
+        dir("test-cypress/") {
+          sh "echo //registry.npmjs.org/:_authToken=${NPM_TOKEN} > .npmrc && npm publish"
+        }
+      }
+    }
+  }
+
+  post {
+    success {
```

Comment:
Should we have email notifications on relase?

Reply:
I decided not to send emails, because npm sends on success, but now on second thought I agree that we should send emails for failing releases. I will add this.

Reply:
What do you think, should we send emails always, or on failing builds only?

Reply:
+1 for on fail only

Reply:
Added step to send email on fail